example_compute.py

```diff
@@ -21,14 +21,8 @@
 
 from typing import Type, cast
 
-ec2_cls = get_driver(Provider.EC2)
```

Comment:
Sadly this doesn't work as expected.

This means we need to rely on base / parent class annotations until we have type annotations for all the driver subclasses.

https://github.com/apache/libcloud/pull/1306#issuecomment-593092777